pkg/envtest/server.go

```diff
@@ -229,18 +229,18 @@ func (te *Environment) Start() (*rest.Config, error) {
 
 		if os.Getenv(envAttachOutput) == "true" {
 			te.AttachControlPlaneOutput = true
```

Comment:
@yanggangtony `AttachControlPlaneOutput` is set to `true` if `os.Getenv(envAttachOutput) == "true"`, however it can also be true if `os.Getenv(envAttachOutput) != "true"` and the `AttachControlPlaneOutput` value was set to true.
So, please create a new `if te.AttachControlPlaneOutput {` check that contains all the other checks.